parsers/verilog.c

```diff
@@ -37,8 +37,8 @@
 #include "ptrarray.h"
 
 /*
-*   MACROS
-*/
+ *   MACROS
+ */
```

Comment:
I don't like the style:
```
/*
*
...
*/
```
style. However, this is the standard style in ctags. See other source files.
If you really want to change the style to
```
/*
 *
 ...
 */
```
style, I will not object to the change.

Reply:
The issue was two styles were used.
I did not check other files in ctags and chose the style I was familiar with. Let me go with this style.